politeiawww/cmd/politeiawwwcli/commands/edituser.go

```diff
@@ -1,61 +1,25 @@
 package commands
 
 import (
-	"fmt"
-	"strconv"
-
 	"github.com/decred/politeia/politeiawww/api/v1"
 )
 
 type EditUserCmd struct {
-	Args struct {
-		UserID string `positional-arg-name:"userid" description:"User ID"`
-		Action string `positional-arg-name:"action" description:"Edit user action"`
-		Reason string `positional-arg-name:"reason" description:"Reason for editing the user"`
-	} `positional-args:"true" required:"true"`
+	MyProposalNotifications      *uint64 `long:"myproposalnotifications" optional:"true" description:"Whether to notify via email about my proposals"`
+	RegularProposalNotifications *uint64 `long:"regularproposalnotifications" optional:"true" description:"Whether to notify via email about others' proposals"`
+	AdminProposalNotifications   *uint64 `long:"adminproposalnotifications" optional:"true" description:"Whether to notify via email about proposals that require admin attention"`
 }
 
 func (cmd *EditUserCmd) Execute(args []string) error {
-	EditActions := map[string]v1.UserEditActionT{
-		"expirenewuser":       1,
-		"expireupdatekey":     2,
-		"expireresetpassword": 3,
-		"clearpaywall":        4,
-		"unlock":              5,
-		"deactivate":          6,
-		"reactivate":          7,
-	}
-
-	// Parse edit user action.  This can be either the numeric
-	// action code or the human readable equivalent.
-	var action v1.UserEditActionT
-	a, err := strconv.ParseUint(cmd.Args.Action, 10, 32)
-	if err == nil {
-		// Numeric action code found
-		action = v1.UserEditActionT(a)
-	} else if a, ok := EditActions[cmd.Args.Action]; ok {
-		// Human readable action code found
-		action = a
-	} else {
-		return fmt.Errorf("Invalid useredit action.  Valid actions are:\n  " +
-			"expirenewuser         expires new user verification\n  " +
-			"expireupdatekey       expires update user key verification\n  " +
-			"expireresetpassword   expires reset password verification\n  " +
-			"clearpaywall          clears user registration paywall\n  " +
-			"unlock                unlocks user account from failed logins\n  " +
-			"deactivate            deactivates user account\n  " +
-			"reactivate            reactivates user account")
-	}
-
 	// Setup request
 	eu := &v1.EditUser{
-		UserID: cmd.Args.UserID,
-		Action: action,
-		Reason: cmd.Args.Reason,
+		MyProposalNotifications:      cmd.MyProposalNotifications,
+		RegularProposalNotifications: cmd.RegularProposalNotifications,
+		AdminProposalNotifications:   cmd.AdminProposalNotifications,
 	}
```

Comment:
Right now you have three proposal notification fields on the User object: `MyProposalNotifications`, `RegularProposalNotifications`, and `AdminProposalNotifications`. Why not just have a single `ProposalNotifications` field with designated notification bits for the three subcategories?  That seems simpler.